### DISCUSSION TITLE
Google Maps

### DISCUSSION THREAD

**Discussion by bergemann9579** [SafeExamBrowser/seb-win-refactoring#573] [Q&A]
Hi,

i would like to Filter some Websites, what the SEB does great. The only Site i always get the error "This Site is blocked". I tried several URL f.eks. https://google.de/maps/ or http://www.google.de/maps or google.de/maps none works. Outside from SEB they all works fine.

What can be the reason for this?

Thanks for the help.

**Comment by dbuechel** [comment]
You can inspect the inner workings of SEB's request filter in real time via the application log, which needs to be (temporarily) enabled in the configuration under "Security" > "Allow access to application log" (see also https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane).

**Comment by bergemann9579** [comment]
Thanks fpor the answer. I did so, but i still cant see what the reason for the blocking is. I attach a pic of the log.
![IMG_0469](https://user-images.githubusercontent.com/126267435/221344704-72b3dae7-8d0f-49af-a068-348289ad3a3d.jpg)

**Comment by bergemann9579** [comment]
I just found out, that if i go to google and accept the cookie, i can open google maps after that. if i dont do it, i still cant open google maps. is there any solution for this?

Thanks.

**Comment by bergemann9579** [comment]
I found a solution, i just dont delete the cache for now.

Thanks for the help.
